pyecharts/constants.py

```diff
@@ -2,58 +2,12 @@
 # coding=utf-8
 from __future__ import unicode_literals
 
-
 DEFAULT_HOST = 'https://chfw.github.io/jupyter-echarts/echarts'
 
 CONFIGURATION = dict(
     HOST='/nbextensions/echarts'
 )
 
-DEFAULT_JS_LIBRARIES = dict(
```

Comment:
This part of the configuration lives in jupypter-echarts now. In this way, the inclusion of future maps will be streamlined, meaning no changes required but simply update jupyter-echarts submodule as of now.